src/core/sentinel.ts

```diff
@@ -0,0 +1,35 @@
+// @ts-ignore
+import sentinelWasm from "./sentinel.wasm";
+
+declare const sentinelWasm: Uint8Array;
+
+const sentinelInstancePromise: Promise<WebAssembly.Instance | undefined> =
+  (async function () {
+    const isIOS =
+      globalThis.navigator && /iPad|iPhone|iPod/.test(navigator.platform);
```

Comment:
this check was not successful in iPad, so probably need to be updated:

https://github.com/pyodide/pyodide/blob/main/cpython/patches/0009-Fix-iPad-detection-in-wasm-gc.patch

Reply:
Oops I copied it from the wrong branch of Python. Thanks!